data/blog/wolvctf-2023/wannaflag.mdx

```diff
@@ -0,0 +1,335 @@
+---
+
+![WannaFlag Banner](/static/images/wolvctf-2023/banner.svg)
+
+<TOCInline toc={props.toc} asDisclosure open={false} exclude={'Intro'} />
+
+### Intro
+
+Over the last weekend, I played in WolvSec’s second CTF iteration with [Project Sekai](https://sekai.team/)—[WolvCTF 2023](https://ctftime.org/event/1866). We placed first in the open division, and throughout the solving process I became intrigued by a specific series of challenges placed under the OSINT category: **WannaFlag**. Telling a story of a supposed ransomware group which had been terrorizing the CTF community for the past several months, these series of challenges offered an opportunity for players to track down this group’s means of operation. Ultimately, the goal was to find WannaFlag’s kingpin through all possible methods.
+
+Project Sekai was the first to blood the entire series. Here was our thought process, notes, and conclusions.
+
+---
+
+## WannaFlag I: An Introduction
+
+> Welcome to WolvCTF’s OSINT Category! We have a bunch of great OSINT lined up, assuming nothing goes wrong hahahhahahhahah but why would it?  
+> For this challenge, find where the image was taken, and look at the Google Maps reviews!  
+> **Note**: Flags can be found in standard format `wctf{...}` for ALL OSINT challenges  
+> **Attachments**: [image.png](/static/images/wolvctf-2023/image.png)
+
+We’re first given a little bit of a warmup: find the location of the following object, and to view its Google Reviews:
+
+![image.png](/static/images/wolvctf-2023/image.png)
+
+Simple task! These types of challenges, often called GEOINT (geospacial intelligence), can be trivial if there is a landmark object situated within the image—in this case, we have some public art resembling a black cube. We can use [Google Lens](https://lens.google) to identify it:
+
+![lens.png](/static/images/wolvctf-2023/lens.png)
+
+Looks like Google’s given us a hit: this is “The Cube,” a public art installation in the University of Michigan. Let’s take a look at the Google Reviews:
+
+![reviews.png](/static/images/wolvctf-2023/reviews.png)
+
+This `netcat wanna-flag-i dot wolvctf dot io one three three seven` can be converted to a command: `nc wanna-flag-i.wolvctf.io 1337`. Let’s connect to this server to see what it has to say:
+
+![netcat.gif](/static/images/wolvctf-2023/netcat.gif)
+
+Well... that was unexpected. Let’s first digest what in the world just happened to this netcat:
+
+- Our flag is cut off midway, and WannaFlag supposedly “pwns” the server
+- We’re told to pay 500,000 Goerli to the wallet `0x08f5AF98610aE4B93...`
+- This hashtag `#YourFlagsBelongToUs` is spammed everywhere throughout the message
+- The group tells “John OSINTs” to leave them alone on Twitter
+
+There’s a small connection here, but it’s not immediately obvious. We can actually search for this hashtag on Twitter, revealing a tweet from none other than `@JohnOSINT_` himself:
+
+<Tweet tweetId="1634680651768270848" />
```

Comment:
Include plain text tweet in blockquote. Same below. See [nmpz](https://github.com/jktrn/sekai.team/blob/master/data/blog/idekctf-2022/nmpz.mdx?plain=1#L524) for example.

Reply:
Resolved with [`65f32fb` (#35)](https://github.com/project-sekai-ctf/sekai.team/pull/35/commits/65f32fbc8eee1ff4844bf9a159d357c1234a8870).